ops.go

```diff
@@ -77,19 +77,45 @@ func Tail[T any](num int, arr []T) []T {
 	return arr[len(arr)-num : len(arr)-1]
 }
 
-// Drop drops up to the first `num` elements.
+// Drop allocates a new slice, with the first `num` elements dropped.
 func Drop[T any](num int, arr []T) []T {
 	if len(arr) < num {
 		return []T{}
 	}
-	return arr[num:]
+
+	newLen := len(arr) - num
+	slice := make([]T, 0, newLen)
+	slice = append(slice, arr[num:]...)
+
+	return slice
 }
 
-// DropString drops up to the first `num` runes of a string.
+// DropString allocates a new string, with the first `num` bytes of a string dropped.
 func DropString(num int, what string) string {
 	if len(what) < num {
 		return ""
 	}
+
+	newLen := len(what) - num
+	buffer := make([]byte, 0, newLen)
+	buffer = append(buffer, what[num:]...)
+
+	return string(buffer)
```

Comment:
I've been thinking on how to run this on runes instead of bytes. Maybe it's a whole different type, I forget.

Reply:
We would have to manually iterate over each rune until the index is where we want to copy from. Maybe this could be DropRunes, SkipRunes?